features/canvas-2d.yml.dist

```diff
@@ -0,0 +1,259 @@
+  - api.CanvasRenderingContext2D.putImageData
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "2"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "3"
+  #   firefox_android: "4"
+  #   safari: "4"
+  #   safari_ios: "3.2"
+  - api.CanvasRenderingContext2D.setTransform
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "2"
+  #   chrome_android: "18"
+  #   edge: "12"
+  #   firefox: "3.5"
+  #   firefox_android: "4"
+  #   safari: "4"
+  #   safari_ios: "3.2"
+  - api.CanvasRenderingContext2D.createImageData
+  - api.CanvasRenderingContext2D.fillText
+  - api.CanvasRenderingContext2D.font
+  - api.CanvasRenderingContext2D.measureText
+  - api.CanvasRenderingContext2D.strokeText
+  - api.CanvasRenderingContext2D.textAlign
+  - api.CanvasRenderingContext2D.textBaseline
+
+  # baseline: high
+  # baseline_low_date: 2015-07-29
+  # baseline_high_date: 2018-01-29
+  # support:
+  #   chrome: "23"
+  #   chrome_android: "25"
+  #   edge: "12"
+  #   firefox: "27"
+  #   firefox_android: "27"
+  #   safari: "7"
+  #   safari_ios: "7"
+  - api.CanvasRenderingContext2D.getLineDash
+  - api.CanvasRenderingContext2D.lineDashOffset
+  - api.CanvasRenderingContext2D.setLineDash
+
+  # baseline: high
+  # baseline_low_date: 2016-08-02
+  # baseline_high_date: 2019-02-02
+  # support:
+  #   chrome: "31"
+  #   chrome_android: "31"
+  #   edge: "13"
+  #   firefox: "48"
+  #   firefox_android: "48"
+  #   safari: "9"
+  #   safari_ios: "9"
+  - api.CanvasRenderingContext2D.ellipse
+
+  # baseline: high
+  # baseline_low_date: 2016-08-02
+  # baseline_high_date: 2019-02-02
+  # support:
+  #   chrome: "37"
+  #   chrome_android: "37"
+  #   edge: "14"
+  #   firefox: "32"
+  #   firefox_android: "32"
+  #   safari: "8"
+  #   safari_ios: "8"
+  - api.CanvasRenderingContext2D.drawFocusIfNeeded
+
+  # baseline: high
+  # baseline_low_date: 2017-04-05
+  # baseline_high_date: 2019-10-05
+  # support:
+  #   chrome: "30"
+  #   chrome_android: "30"
+  #   edge: "15"
+  #   firefox: "51"
+  #   firefox_android: "51"
+  #   safari: "9.1"
+  #   safari_ios: "9.3"
+  - api.CanvasRenderingContext2D.imageSmoothingEnabled
+
+  # baseline: high
+  # baseline_low_date: 2020-01-15
+  # baseline_high_date: 2022-07-15
+  # support:
+  #   chrome: "26"
+  #   chrome_android: "26"
+  #   edge: "79"
+  #   firefox: "19"
+  #   firefox_android: "19"
+  #   safari: "7"
+  #   safari_ios: "7"
+  - api.CanvasRenderingContext2D.isPointInStroke
+
+  # baseline: high
+  # baseline_low_date: 2020-01-15
+  # baseline_high_date: 2022-07-15
+  # support:
+  #   chrome: "31"
+  #   chrome_android: "31"
+  #   edge: "79"
+  #   firefox: "36"
+  #   firefox_android: "36"
+  #   safari: "10.1"
+  #   safari_ios: "10.3"
+  - api.CanvasRenderingContext2D.resetTransform
+
+  # ⬇️ Same status as overall feature ⬇️
```

Comment:
I'd say it's very weird to say that 2D canvases have only been supported since 2020, when IE could do a bunch of this stuff. I think we need a `compute_from` such that the Baseline low date is 2018-01-29, even though it'll be a bit arbitrary which key(s) you pick.

Reply:
100% agree, didn't specify one yet until we get this a little closer. As you said, it's somewhat arbitrary.